plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/FileSystemExchangeModule.java

```diff
@@ -53,11 +55,13 @@ protected void setup(Binder binder)
         if (scheme == null || scheme.equals("file")) {
             binder.bind(FileSystemExchangeStorage.class).to(LocalFileSystemExchangeStorage.class).in(Scopes.SINGLETON);
         }
-        else if (ImmutableSet.of("s3", "s3a", "s3n").contains(scheme)) {
```

Comment:
add entry to the release notes. Is there some doc to be updated?

Reply:
I have updated https://github.com/trinodb/trino/pull/12472 accordingly